lib/http/form_data/multipart.rb

```diff
@@ -24,7 +26,7 @@ def initialize(data)
       #
       # @return [String]
       def content_type
-        "multipart/form-data; boundary=#{@boundary}"
+        "multipart/form-data; boundary=#{boundary}"
```

Comment:
But we have tests for those, right? :D I mean I got your point. It sounds reasonable to me, but accessing instance variables is a bit faster (no big difference in real world though). :D Thus if we have tests to cover that `content_type` returns correct value, it's safe to use `@boundary`  here :D